custom/icds_reports/management/commands/sql_scripts/urgent_pmo_datapull.sql

```diff
@@ -0,0 +1,85 @@
+select sum(pse_attended_21_days) as child_pse,sum(lunch_count_21_days) as child_hcm,sum(rations_21_plus_distributed) as child_thr, sum(weighed_and_height_measured_in_month) as height_weight_measured_in_month, sum(bf_at_birth) as bf_at_birth, sum(born_in_month) as born_in_month, sum(cf_initiation_in_month) as cf_initiation_in_month, sum(cf_initiation_eligible) as cf_initiation_eligible, sum(nutrition_status_weighed) as nutrition_status_weighed, sum(wasting_severe) as wasting_severe, sum(wasting_moderate) as wasting_moderate, sum(weighed_and_height_measured_in_month) as weighed_and_height_measured_in_month, sum(ebf_in_month) as ebf_in_month, sum(nutrition_status_moderately_underweight)+sum(nutrition_status_severely_underweight) as underweight_children, (sum(fully_immunized_on_time)+sum(fully_immunized_late))/sum(fully_immunized_eligible) as immunization, SUM(wer_eligible) as wer_eligible_child_health from "agg_child_health" inner join awc_location_local on "agg_child_health".state_id = awc_location_local.state_id and "agg_child_health".aggregation_level=awc_location_local.aggregation_level where "agg_child_health".month='{month}' and "agg_child_health".aggregation_level=1 and awc_location_local.state_is_test is distinct from 1;
```

Comment:
Where ever you are calculating percentage. you can convert one of the numerator or denominator as float and multiple by 100 otherwise it will come as 0.

Reply:
updated!